source/features/first-published-tag-for-merged-pr.tsx

```diff
@@ -118,14 +122,8 @@ void features.add(import.meta.url, {
 		pageDetect.isOpenPR,
 		canCreateRelease,
 	],
-	additionalListeners: [
-		onPrMerge,
-	],
-	onlyAdditionalListeners: true,
 	deduplicate: false,
-	init() {
-		void addReleaseBanner();
-	},
```

Comment:
I thought this feature was affected because it relied on the old behavior, but not really.

However it would greatly benefit from being rewritten with the new observer instead of using additional listeners, also because we have two separate UI pieces, one of which needs to be applied multiple times.

Related:

- https://github.com/refined-github/refined-github/issues/5895
- https://github.com/refined-github/refined-github/issues/5222#issuecomment-1242931510

Reply:
Also this would help rewrite `pr-branch-auto-delete`:

- https://github.com/fregante/delegate-it/issues/34